src/App.js

```diff
@@ -1,18 +1,28 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Header from './containers/Header/Header';
+import RestaurantsAndMap from './containers/RestaurantsAndMap/RestaurantsAndMap';
+import axios from './axios';
```

Comment:
I suggest using `https://github.com/github/fetch`. It polyfills a browser capability if it's missing and sort of makes Axios redundant.